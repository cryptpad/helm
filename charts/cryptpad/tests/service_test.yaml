suite: test service
templates:
  - service.yaml
tests:
  - it: should set values properly
    set:
      service.type: ClusterIP
      service.name: tcp
    release:
      name: release
    asserts:
      - isKind:
          of: Service
#        template: service.yaml
      - matchSnapshot:
          path: spec
#        template: service.yaml
  - it: should have same labels as Deployment
    set:
      service.type: ClusterIP
      service.externalPort: 81
      service.internalPort: 8081
      service.name: tcp
    release:
      name: release
    asserts:
      - equal:
          path: spec.selector
          value:
           app.kubernetes.io/instance: release
           app.kubernetes.io/name: cryptpad