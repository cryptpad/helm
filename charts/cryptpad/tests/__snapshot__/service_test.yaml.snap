should set values properly:
  1: |
    ports:
      - name: tcp
        port: 80
        protocol: TCP
        targetPort: 8080
    selector:
      app.kubernetes.io/instance: release
      app.kubernetes.io/name: cryptpad
    sessionAffinity: ClientIP
    type: ClusterIP
